### PR TITLE
[#374] Add component: breadcrumbs

### LINKED ISSUE #374: Add component: breadcrumbs
Need a breadcrumb component

### PR DESCRIPTION
**Pull request by planktonic**
Fixes #374 

Adds two breadcrumbs examples

Looks like:

Simple, with all text 

Screenshot 2021-02-17 at 17 17 52

With an icon

Screenshot 2021-02-17 at 17 17 57

### CODE REVIEW

File: scss/bitstyles/objects/icon/settings.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 $bitstyles-icon-sizes: (
   's':     spacing('xs'),
   'm':     spacing('m'),
-  'l':     spacing('xl')
+  'l':     spacing('l')
```
Comment (klappradla) [inline scss/bitstyles/objects/icon/settings.scss:4]: 😉

File: scss/bitstyles/tools/_icon.scss [hunk 2]
```diff
@@ -4,7 +4,12 @@
...
   font-size: 1em;
   vertical-align: middle;
+  flex-shrink: 0;
   outline: none;
-  transition: fill $bitstyles-transition-duration $bitstyles-transition-easing, stroke $bitstyles-transition-duration $bitstyles-transition-easing, background-color $bitstyles-transition-duration $bitstyles-transition-easing, transform $bitstyles-transition-duration $bitstyles-transition-easing;
+  transition:
+    fill $bitstyles-transition-duration $bitstyles-transition-easing,
+    stroke $bitstyles-transition-duration $bitstyles-transition-easing,
+    background-color $bitstyles-transition-duration $bitstyles-transition-easing,
+    transform $bitstyles-transition-duration $bitstyles-transition-easing;
```
Comment (klappradla) [inline scss/bitstyles/tools/_icon.scss:13]: Nice 🧹

File: scss/bitstyles/components/breadcrumbs.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Components/Breadcrumbs" />
+
+# Breadcrumbs
+
+A list of links charting the user’s place in the site structure. The title of the current page should be in an `<h1>` on the page, so it isn’t needed here.
+
+<Canvas isColumn>
+  <Story name="Breadcrumbs">
```
Comment (klappradla) [inline scss/bitstyles/components/breadcrumbs.stories.mdx:10]: For some reason I see this example jumping (even though there's no animation on it) 🤔 

Kapture 2021-02-18 at 11 20 47

**Review by klappradla — APPROVED** [review]
Vale 👍 

The non-icon example behaves a bit weird for me, but aside that ✨

File: scss/bitstyles/components/breadcrumbs.stories.mdx (hunk 3 above)
Comment (planktonic) [reply scss/bitstyles/components/breadcrumbs.stories.mdx:10]: lolwut 😅 Thanks, I’ll look into it!

Comment (planktonic) [reply scss/bitstyles/components/breadcrumbs.stories.mdx:10]: Ok that’s weird  🙈It does the same with the second example if you view that one in the canvas mode then switch to docs. In Firefox it’s transitioned, in safari it’s not. I’ll put that down to a storybook bug for now, and reinvestigate later: https://github.com/bitcrowd/bitstyles/projects/1#card-55195339

**Pull request merged**